src/code/z_room.c

```diff
@@ -609,8 +609,8 @@ u32 Room_SetupFirstRoom(PlayState* play, RoomContext* roomCtx) {
                                              : play->spawnList[play->spawn].room;
 
     // Load into a room for the first time.
-    // Since curRoom was initialized to (room -1, segment NULL) in Play_InitScene, the previous room
-    // the nulled state when this function completes
+    // Since curRoom was initialized to `room` = -1 and `segment` = NULL in Play_InitScene, the previous room
+    // will also be initialized to the nulled state when this function completes.
```

Comment:
if were gonne phrase it like this id prefer backticks around the whole expression since its all code 
`room = -1` and `segment = NULL`

But I wont hold it up cause of this